lte/gateway/deploy/roles/service_aliases/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Create convenient aliases for magma services
+  lineinfile:
+    dest: /home/{{ ansible_user }}/.bashrc
+    state: present
+    line: "{{ item }}"
+  with_items:
+    - "alias magma-start='sudo service magma@magmad start'"
+    - "alias magma-stop='sudo service magma@* stop'"
+    - "alias magma-restart='sudo service magma@* stop && sudo service magma@magmad start'"
```

Comment:
Could we include sctpd and magma_dp@envoy in these commands?

Reply:
I am not completely sure what the implications of that would be. These are the commands that the Makefile had and that should be used for the integration tests. Sctpd and envoy are already started if magmad is started as they are part of/depend on MME/Pipelined.

Reply:
But this alias is not use by the ci is it?

Reply:
No the alias is not used by ci as we decided to use the commands directly. However I fail to see the advantage of additionally specifying the sctpd and envoy services since their starts/stops/restarts are already handled by the service definitions. Moreover the behaviour of these commands should be consistent with the behaviour of the Make targets.

Reply:
Afaik sctpd is only started with the magma service but not stopped/restarted.

It is left running as a sctpd restart would require all subscribers to reattach. This is important for prod systems but not for the test vm.

However if you want to stop/restart all services for development purposes it makes much more sense to include sctpd as well.

Usally a restart of sctpd without magma will lead the test to get stuck in "S1 setup" step.

I don't care about the envoy it is only used in 2-3 tests anyway and have never been required to restart it.

Reply:
Changed it to:
```  
- "alias magma-stop='sudo service magma@* stop && sudo service sctpd stop && sudo service magma_dp@envoy stop'"
- "alias magma-restart='magma-stop && magma-start'"
- "alias magma-status='sudo service magma* status'"
```
Magma-status now also includes the `magma_dp@envoy`. It is not possible to include `sctpd` in the same status command as service ... status` opens a kind of window viewer.